release-content/migration-guides/render_startup.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Many render resources now initialized in `RenderStartup`
+pull_requests: [19841, 19926, 19885, 19886, 19897, 19898, 19901]
+---
+
+Many render resources are **no longer present** during `Plugin::finish`. Instead they are
+initialized during `RenderStartup` (which occurs once the app starts running). If you only access
+the resource during the `Render` schedule, then there should be no change. However, if you need one
+of these render resources to initialize your own resource, you will need to convert your resource
+initialization into a system.
+
+The following are the (public) resources that are now initialized in `RenderStartup`.
+
+- `CasPipeline`
+- `FxaaPipeline`
+- `SmaaPipelines`
+- `TaaPipeline`
+- `BoxShadowPipeline`
+- `GradientPipeline`
+- `UiPipeline`
+- `UiMaterialPipeline<M>`
+- `UiTextureSlicePipeline`
+
+The vast majority of cases for initializing render resources look like so:
```

Comment:
Looked like this in Bevy 0.16?

Reply:
Done. Mentioned in both the release notes and migration code that the code looked like this in 0.16.